content/components/selectpanel.mdx

```diff
@@ -79,14 +55,7 @@ For example setting the wrong assignee on an pull request poses the risk of send
   src="https://github.com/primer/design/assets/980622/bfaa7eca-9a12-432d-9184-38f024a7cd32"
 />
 
-The multi-select list allows you to choose multiple items from a list and has additional features like **select all** or **indeterminate selections**.
-
-The **select all** option at the top of the list allows for quickly deselecting or selecting all values.
-However, when implementing search functionality, this option becomes unavailable.
-
-We highly recommend utilizing the search functionality instead of retrieving extensive lists from our API endpoints to enable the select all feature. By doing so, you can avoid potential performance issues, especially considering that this component lacks virtualization capabilities.
-
-The **indeterminate selections** option is useful for performing bulk actions on lists. For example, if you have 20 selected items, but some of them have different labels attached, an indeterminate checkbox (represented by a horizontal line) is shown for labels that are not set on all items.
+This option allows choosing multiple items and includes features like "select all" for quick selection/deselection and "indeterminate selections" for bulk actions. Implementing a search function is recommended over displaying extensive lists to avoid performance issues.
```

Comment:
Brought back the more detailed description but changed the order of it so that search comes first and "Select all" last. That way it's sorted by priority respectively likelihood of adoption scenario. Also swapped the examples in the image above that you provided me 🙇 to accompany that.

https://primer-3de09d22a0-26441320.drafts.github.io/components/selectpanel#list-multi-select

![Screenshot of the rendered section with the changes described above](https://github.com/primer/design/assets/3369400/be2353c3-b333-46ab-ac21-924149c3203c)